src/apis/stockx/bids/index.js

```diff
@@ -49,10 +51,10 @@ export default class Bids extends Base {
 
     checkStatus(res);
 
-    const { body } = res;
-    const { PortfolioItem: { chainId, skuUuid } } = body;
-    
-    return { chainId, skuUuid };
+    const body = JSON.parse(res.body);
```

Comment:
Same as the other PR. I'm not sure the `JSON.parse()` is needed. Please correct me if I'm wrong!